.github/workflows/release.yml

```
@@ -0,0 +1,55 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        GOOS: [ darwin, linux, windows ]
+        GOARCH: [ amd64 ]
+        include:
+          - GOOS: darwin
+            GOARCH: amd64
+            suffix: darwin-amd64
+          - GOOS: windows
+            GOARCH: amd64
+            suffix: windows-amd64.exe
+          - GOOS: linux
+            GOARCH: amd64
+            suffix: linux-amd64
+          - GOOS: linux
+            GOARCH: arm
+            suffix: linux-armv7
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
```

```suggestion
      - uses: actions/setup-go@v2
        with:
          go-version: '1.15'      
```

@aspacca you want a specific version?

---

latest would be fine. it should have been upgraded to `1.16` on travis as well